### ISSUE TITLE
Batching default queue limit seems too small

### ISSUE THREAD

**Issue by srollinet** [whir1/serilog-sinks-graylog#42]
Hi,

It took me a moment to figure out why a lot of logs were missing in Graylog and I finally found that this is because of the default queue size limit (10)

because of that, if you send more than 10 log events, only the first 10 are added in the queue and the other events are lost. 

Is there a reason for a so small value? If I take another implementation (https://github.com/serilog/serilog-sinks-azuretablestorage/blob/master/src/Serilog.Sinks.AzureTableStorage/Sinks/AzureTableStorageWithProperties/AzureBatchingTableStorageWithPropertiesSink.cs) the don't use the constructor with the queueLimit parameter. Which means no size limit.

**Comment by srollinet** [comment]
By the way, it seem I cannot override the value using the configuration.

Here is my configuration
```
      {
        "Name": "Graylog",
        "Args": {
          "minimumLogEventLevel": "Information",
          "hostnameOrAddress": "http://localhost",
          "port": "12201",
          "transportType": "Http",
          "queueLimit": 1000
        }
      }
```

but in the runtime, queueLimit is still 10

**Comment by whir1** [comment]
> Is there a reason for a so small value?
No, i haven't any reason it's just default value.

Configuration not working because i'm forget pass parameters to the LoggerConfigurationGrayLogExtensions =(
Fixed in version 1.0.6 with default value of queueLimit 1000

**Comment by srollinet** [comment]
Great and thanks for the reactivity !
